frontend/src/views/GCredentials.vue

```diff
@@ -171,15 +167,11 @@ SPDX-License-Identifier: Apache-2.0
             absolute
           >
             <template #activator="{ props: menuProps }">
-              <v-tooltip location="top">
-                <template #activator="{ props: tooltipProps }">
-                  <v-btn
-                    v-bind="mergeProps(menuProps, tooltipProps)"
-                    icon="mdi-plus"
-                  />
-                </template>
-                Create Secret Credential
-              </v-tooltip>
+              <v-btn
+                v-tooltip:top="'Create Secret Credential'"
```

Comment:
```suggestion
                v-tooltip:top="'Create DNS Credential'"
```
there is also at least one other place where this has to be fixed